src/tests/pages/Feedbackscreen.test.js

```diff
@@ -34,3 +36,86 @@ it("check if textarea displays text", () => {
 
   expect(textarea.value).toBe("feedback");
 });
+
+test("check if textarea is cleared after sendFeedback returns success", async () => {
+  const { getByTestId } = render(
+      <Router>
+        <Feedbackscreen />
+      </Router>
+    ),
+    textarea = getByTestId("feedback-textarea");
+  fireEvent.change(textarea, { target: { value: "fake user input" } });
+
+  // fake positive response from sendFeedback sevice
+  sendFeedback.mockImplementation(() => Promise.resolve(true));
+
+  // click button and wait for mock function to be returned
+  fireEvent.click(getByTestId("feedback-button"));
+  await waitFor(() => expect(sendFeedback).toHaveReturned());
+
+  expect(textarea.value).toBe("");
+});
+
+test("check if textarea is not cleared if sendFeedback returns seccess", async () => {
```

Comment:
Typo in 'seccess' and unsure if you wanted to write 'success' and then fake a negative response?

Reply:
thanks, classic copy and paste anti pattern...